app/services/idv/agent.rb

```diff
@@ -1,26 +1,23 @@
 module Idv
   class Agent
-    class << self
-      def proofer_attribute?(key)
-        Idv::Proofer.attribute?(key)
```

Comment:
These `attribute?` functions didn't seem to be used anywhere outside of tests, so I removed them. The most recent reference outside of tests I could find from the git log is where it was removed in https://github.com/18F/identity-idp/commit/19ddf721f9895514708a749634c532fecf9ca09d#diff-2df0b9c9e83aa13bc56217ce31c8040cL129